repo.sh

```diff
@@ -41,7 +41,16 @@ private_repos=(
     "https://github.com/edx/edx-themes.git"
 )
 
+wp_plugin_repos=(
+    "git@github.com:edly-io/edly-wp-plugin.git"
+)
+
+wp_theme_repos=(
+    "git@github.com:edly-io/edly-wp-theme.git"
+)
```

Comment:
Kindly add those plugins too.
1. Elementor
2. Classic Editor
3. Open edX LMS and WordPress integrator
4. Advanced Custom Fields Pro

Reply:
Why do we need the classic editor?

Reply:
Confirmed from Amjad bhai. Will be adding that as well.

Reply:
Done.